final-markup-task/index.html

```diff
@@ -0,0 +1,518 @@
+                <img class="price-item__feature-img" src="./assets/img/check_icon.png" alt="check icon image" />Lorem
+                ipsum dolor sit amet
+              </li>
+              <li class="price-item__feature-item">
+                <img
+                  class="price-item__feature-img"
+                  src="./assets/img/check_icon.png"
+                  alt="check icon image"
+                />Consectetuer adipiscing elit
+              </li>
+              <li class="price-item__feature-item">
+                <img class="price-item__feature-img" src="./assets/img/check_icon.png" alt="check icon image" />Sed diam
+                nonummy
+              </li>
+              <li class="price-item__feature-item">
+                <img class="price-item__feature-img" src="./assets/img/check_icon.png" alt="check icon image" />Nibh
+                euismod tincidunt
+              </li>
+              <li class="price-item__feature-item">
+                <img class="price-item__feature-img" src="./assets/img/check_icon.png" alt="check icon image" />Ut
+                laoreet dolore
+              </li>
+              <li class="price-item__feature-item">
+                <img class="price-item__feature-img" src="./assets/img/check_icon.png" alt="check icon image" />Magna
+                aliquam erat volutpat
+              </li>
+            </ul>
+            <button class="price-item__btn">get started</button>
+            <div class="pricing__item_top-line"></div>
+            <div class="pricing__item_top-corner"></div>
+          </li>
+          <li class="pricing__item price-item">
+            <div class="pricing__item-animation"></div>
+            <h4 class="price-item__header">smart</h4>
+            <h5 class="price-item__price">$75.99 Monthly</h5>
+            <ul class="price-item__feature-list">
+              <li class="price-item__feature-item">
+                <img class="price-item__feature-img" src="./assets/img/check_icon.png" alt="check icon image" />Lorem
+                ipsum dolor sit amet
+              </li>
+              <li class="price-item__feature-item">
+                <img
+                  class="price-item__feature-img"
+                  src="./assets/img/check_icon.png"
+                  alt="check icon image"
+                />Consectetuer adipiscing elit
+              </li>
+              <li class="price-item__feature-item">
+                <img class="price-item__feature-img" src="./assets/img/check_icon.png" alt="check icon image" />Sed diam
+                nonummy
+              </li>
+              <li class="price-item__feature-item">
+                <img class="price-item__feature-img" src="./assets/img/check_icon.png" alt="check icon image" />Nibh
+                euismod tincidunt
+              </li>
+              <li class="price-item__feature-item">
+                <img class="price-item__feature-img" src="./assets/img/check_icon.png" alt="check icon image" />Ut
```

Comment:
HTML will be clean if you will use pseudo element `before` to create marker for the list